lobe_android/app/src/main/java/com/example/test/tflite/ImageClassifier.java

```diff
@@ -26,28 +26,30 @@
 import org.tensorflow.lite.support.common.TensorOperator;
 import org.tensorflow.lite.support.common.ops.NormalizeOp;
 
-/** This TensorFlowLite classifier works with the float MobileNet model. */
-public class ClassifierFloatMobileNet extends Classifier {
+/** This TensorFlowLite classifier works with Lobe exported models. */
+public class ImageClassifier extends Classifier {
 
-  /** Float MobileNet requires additional normalization of the used input. */
+  /**
+   * Requires additional normalization of the used input.
+   */
   private static final float IMAGE_MEAN = 127.5f;
 
   private static final float IMAGE_STD = 127.5f;
 
   /**
-   * Float model does not need dequantization in the post-processing. Setting mean and std as 0.0f
+   * Model does not need dequantization in the post-processing. Setting mean and std as 0.0f
    * and 1.0f, repectively, to bypass the normalization.
    */
   private static final float PROBABILITY_MEAN = 0.0f;
```

Comment:
same here, we aren't using normalization in preprocessing so don't need these

Reply:
I removed it and gave the example to Jake and he confirmed it did not work great at all with his models. So that's why I've left it for now.

Reply:
hmm ok would like to investigate a bit

Reply:
Agreed, will take note in that remaining task.